guides/browser-support.md

```diff
@@ -0,0 +1,171 @@
+# Browser support
+
+> A friendly guide for building cross-browser web components
+
+## Why support?
```

Comment:
Trivial: Two blank lines above H2s in Markdown is preferred. (Below on the "Tips" heading, as well.)